### ISSUE TITLE
Downloads failing

### ISSUE THREAD

**Issue by jw3** [jw3/example-daffodil-vscode#8]
Doesnt seem to attempt the download.

```
> Executing task: ./daffodil-debugger-0.0.4/bin/da-podil <

/bin/bash: ./daffodil-debugger-0.0.4/bin/da-podil: No such file or directory
The terminal process "/bin/bash '-c', './daffodil-debugger-0.0.4/bin/da-podil'" failed to launch (exit code: 127).

Press any key to close the terminal.
```

**Comment by jw3** [comment]
Maybe broken in 6a9ca6aa91e4e80424eb48c0c489436640b394c1

Fix noted in #6 though doesnt appear to be working with v0.0.2

**Comment by jw3** [comment]
Local issue, VS Code indicates the extension needs reloaded between uninstall and reinstall of vsix.
